### PR TITLE
Update mocpy info

### PR DESCRIPTION
**Pull request by fxpineau**
Update mocpy ~"last-update" and~ "maintainer".

- [x] Get astropy/astropy.github.com#477 merged first.

### CODE REVIEW

File: src/static/registry.json [hunk 1]
```diff
@@ -1008,7 +1008,7 @@
                 "testing": "Partial",
                 "devstatus": "Good",
                 "python3": "Yes",
-                "last-updated": "2019-11-04"
+                "last-updated": "2022-02-04"
```
Comment (pllim) [inline src/static/registry.json:1011]: This is pulled from https://github.com/astropy/astropy.github.com/blob/298c6b20f393f3dad8794e511fef685c1fed5dd1/affiliated/registry.json#L996 . If you want to change this, you need to update it first over there.

Also, you don't have to touch `last-updated` as that field is reserved for re-reviewing the package.

Comment (fxpineau) [reply src/static/registry.json]: @pllim  Ok, I just reset the original "last-updated" value.

**Comment by pllim** [comment]
astropy/astropy.github.com#477 is in , though we might want to refresh the whole JSON instead of just this entry? 🤷

**Comment by adrn** [comment]
Sounds good - I'll just copy in the new file

**Pull request closed without merging**